google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -823,23 +823,23 @@ def iter_compute_vpntunnels(self, project_number):
         for vpntunnel in resources:
             yield vpntunnel
 
-    def iter_container_clusters(self, project_number):
-        """Iterate Kubernetes Engine Cluster from Cloud Asset data.
-
-        Args:
-            project_number (str): number of the project to query.
-
-        Yields:
-            dict: Generator of Kubernetes Engine Cluster resources.
-        """
-        resources = self.dao.iter_cai_assets(
-            ContentTypes.resource,
-            'container.googleapis.com/Cluster',
-            '//cloudresourcemanager.googleapis.com/projects/{}'.format(
-                project_number),
-            self.session)
-        for cluster in resources:
-            yield cluster
+    # def iter_container_clusters(self, project_number):
```

Comment:
Can you please make a comment why this is disabled, and when it should be re-enabled?

Reply:
added comment.